frontend/src/types/AssessmentTypes.ts

```diff
@@ -1,10 +1,20 @@
-export type StatusProperty = "DRAFT" | "PUBLISHED" | "ARCHIVED" | "DELETED";
+export enum Status {
+  DRAFT = "Draft",
```

Comment:
in the backend we store the enums capitalized like so https://github.com/uwblueprint/jump-math/blob/staging/backend/models/test.model.ts#L15 can we make sure they're consistent?

Reply:
good catch! updated the enums & then the fe to format properly. also added a general back button component, since i think we'll use it again!